.github/workflows/unassign.yml

```diff
@@ -0,0 +1,29 @@
+name: Unassign outgoing support user
+env:
+  ZENDESK_API_URL: ${{ secrets.ZENDESK_API_URL }}
+  ZENDESK_USERNAME: ${{ secrets.ZENDESK_USERNAME }}
+  ZENDESK_API_KEY: ${{ secrets.ZENDESK_API_KEY }}
+  OPSGENIE_API_KEY: ${{ secrets.OPSGENIE_API_KEY }}
+  FIRST_LINE_DEV_SUPPORT_GROUP_ID: 21306177
+  OPSGENIE_SCHEDULE_ID: "e71d500f-896a-4b28-8b08-3bfe56e1ed76"
+
+on:
+  schedule:
+    - cron: '0 14 * * 1-5'
```

Comment:
This is likely to be confusing when we shift timezones (right now, we're in BST, so +1, i.e., this actually runs at 13:00).

Reply:
You can also use `MON-FRI` for more clarity on what the last numbers mean.

Reply:
Yeah, I've pointed out that these run at UTC. I've come across this before and I'm pretty sure you can't specify what timezone to use in actions.

Reply:
You can't. I think erring for later, rather than earlier is the thing to do for this one, so setting to 15:00 UTC.

Reply:
Yeah, good plan